release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1263,6 +1263,24 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
+
+* Previously, the console UI failed when a *Task* in the Pipeline Builder that was previously installed from the *ArtifactHub* was selected and an error page displayed. With this update, the console UI no longer expects optional data and the console UI no longer fails. (link:https://issues.redhat.com/browse/OCPBUGS-24001[*OCPBUGS-24001*])
+
+* Previously, the *Edit Build* and *BuildRun* options in the *Actions* menu of the Shipwright Plugin did not allow you to edit in the YAML tab. With this update, you can edit in the YAML tab. (link:https://issues.redhat.com/browse/OCPBUGS-23164[*OCPBUGS-23164*])
+
+* Previously, the console searched only for the file name `Dockerfile` in a repository to identify the repository suitable for the `Container` strategy in the Import Flows. Since other containerization tools are available, support for the `Containerfile` file name is now suitable for the `Container` strategy. (link:https://issues.redhat.com/browse/OCPBUGS-22976[*OCPBUGS-22976*])
+
+* Previously, when an unauthorized user opened a link to the console that contains path and query parameters, and they were redirected to a login page, the query parameters did not restore after the login was successful. As a result, the user needed to restore the search or click the link to the console again. With this update, the latest version saves and restores the query parameters similar to the path. (link:https://issues.redhat.com/browse/OCPBUGS-22199[*OCPBUGS-22199*])
+
+* Previously, when navigating to the *Create Channel* page from the *Add* or *Topology* view, the default name as `Channel` is present, but the *Create* button is disabled with `Required` showing under the name field. With this update, if the default channel name is added then the `Required` message will not display when clicking the *Create* button. (link:https://issues.redhat.com/browse/OCPBUGS-19783[*OCPBUGS-19783*])
+
+* Previously, there were similar options to choose from when using the quick search function. With this update, the *Source-to-image* option is differentiated from the *Samples* option in the *Topology* quick search. (link:https://issues.redhat.com/browse/OCPBUGS-18371[*OCPBUGS-18371*])
+
+* Previously, when {serverless-product-name} Operator was installed and the Knative (Kn) serving instance had not been created, then when navigating to the *Global configuration* page from *Administration* -> *Cluster Settings* and clicking *Knative-serving* a `404 page not found` error was displayed. With this update, before adding *Knative-serving* to the *Global configuration*, a check is in place to determine if a Knative serving instance is created. (link:https://issues.redhat.com/browse/OCPBUGS-18267[*OCPBUGS-18267*])
```

Comment:
When reviewing a PR targeting [4.15 RN](https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html-single/release_notes/index#ocp-4-15-bug-fixes), I noticed that {serverless-product-name} is rendering as is, instead of the product name.

<img width="730" alt="Screenshot 2025-01-02 at 5 25 10 PM" src="https://github.com/user-attachments/assets/45c8b4db-c21d-4746-a806-be1fbac82c5c" />